vignettes/tables.Rmd

```diff
@@ -0,0 +1,99 @@
+---
+title: "Summary Table Creation"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{approved}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(timber)
+```
+
+In this section, we write a script to create a risk-difference table from the 
+[PHUSE Test Data Factory ADaM dataset](https://github.com/phuse-org/TestDataFactory), 
+using the [`Typlr`](https://atorus-research.github.io/Tplyr/) package. We use 
+the `timber` package to generate a log upon the script execution for 
+reproducibility and traceability.
+
+## 1. Load the packages
+First, we load the packages that we will use to generate the risk-difference 
+table.
+```{r}
+library(haven)
+library(magrittr)
+library(Tplyr)
+```
+
+## 2. Load the data
+Then we load the Adverse Event Analysis Dataset from the 
+[PHUSE Test Data Factory ADaM dataset](https://github.com/phuse-org/TestDataFactory). 
+The data are stored in a transport file format and we use the 
+[`haven`](https://haven.tidyverse.org) package to load them.
+```{r}
+adae <- read_xpt("adae.xpt")
+```
+
+## 3. Build the risk-difference table
+Finally, we build the risk-difference table, using the 
+[`Typlr`](https://atorus-research.github.io/Tplyr/) package.
+```{r}
+t <- tplyr_table(adae, TRTA) %>% 
+  add_layer(
+    group_count(AEDECOD) %>% 
```

Comment:
@thomasp-b if you updated this to be a variable with less options like RACE the warnings would be less as there would be less times the test is run.  I think that would be a good update and then we could show the warnings and it would not be as cumbersome

Reply:
Thank you, @kodesiba. I used your suggestion in the latest commit.